vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -102,10 +123,39 @@ $( document ).ready(function() {
 	$('#spaceLinkCreationModal.draggable>.modal-dialog>.modal-content>.modal-header').css('cursor', 'move');
 	
 	$('#spaceLinkRotation').change(function() {
-		$('#arrow').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
+		$('#icon').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
 	});
 	
+	function makeItVisible(payload) {
```

Comment:
payload is typically used for post requests or similar actions, it's not the best name here.